v2/cache_utils/cache.go

```diff
@@ -0,0 +1,41 @@
+package cache_utils
```

Comment:
nit: go styleguide is to avoid using `_`

Reply:
done